src/components/input-field/input-field.scss

```diff
@@ -35,6 +35,29 @@ $mdc-text-field-fullwidth-bottom-line-color: $lime-text-field-bottom-line-color;
     @include mdc-states-focus-opacity(0, true);
 }
 
+label {
+    &.mdc-text-field {
+        transition: background-color 0.2s ease;
+
+        &:not(.mdc-text-field--disabled) {
+            // this has to be written this way to be able to override MD styles
+            background-color: transparent;
+
+            &:hover {
+                background-color: #f1f1f3;
```

Comment:
If the point of this PR is to make input fields look good on backgrounds other than white, we can't use a solid hover colour.

Reply:
You're absolutely right about not having solid colors. But colors aren't things we have really decided yet. At some point of time, we need to go through all colors and start a reform which takes into account the dark mode as well. It's gonna be a very complex task and requires a lot of decisions and tests and checks. First thing to do there is to decide for a couple of color palettes, and where and how to use them. We're having a separate discussion about them in the redesign and marketing teams right now.

The point of this PR is to remove the initial background color from the input fields (when they're not focused or active) so that we can have them on backgrounds with other colors.

For active and hover states, we need other good conventions (even perhaps shared with other elements such as radio buttons and checkboxes). For now it's simply too abstract to think about and design for. Especially because there are other indicators (the accented bottom border) to show where a field is focused or not. So I think we can let go of it

Reply:
Ok, as long as you've made a conscious decision, and didn't just overlook it, I'm happy 🙂